conf/conf.go

```diff
@@ -2,29 +2,45 @@
 package conf
 
 import (
-	"github.com/logrusorgru/aurora"
 	"log"
 	"os"
 	"strconv"
 	"strings"
 	"sync"
+
+	aurora "github.com/logrusorgru/aurora/v3"
 )
 
 const defaultInstanceURL = "https://ega.elixir.no"
+const defaultTSDFileAPIVersion = "v1"
```

Comment:
Wouldn't it be better if the `defaultTSDFileAPIVersion`  be parameterized?

Reply:
@a-ghanem 
I have disscussed it with @kjellp . I actually developed it at the first place as you sugggested here @a-ghanem . But he said that it wont change in the future. so it is better to have it as a constant.